src/Microsoft.Azure.SignalR.Common/ServiceConnectionBase.cs

```diff
@@ -119,6 +126,21 @@ public async virtual Task WriteAsync(ServiceMessage serviceMessage)
 
         protected abstract Task OnMessageAsync(ConnectionDataMessage connectionDataMessage);
 
+        protected Task OnServiceErrorAsync(ServiceErrorMessage serviceErrorMessage)
+        {
+            if (!string.IsNullOrEmpty(serviceErrorMessage.ErrorMessage))
+            {
+                // When receives service error message, we suppose server -> service connection doesn't work,
```

Comment:
@davidfowl After discussed offline, we renamed the message name and changed to log rather than throw exceptions in `ProcessIncomingAsync`